microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools.md

```diff
@@ -0,0 +1,242 @@
+---
+description: The More Tools button, in-context documentation to get started using the DevTools extension, increased support for screen readers in the Console, and more.
+title: What's new in DevTools (Microsoft Edge 92)
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 05/25/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+# What's New In DevTools (Microsoft Edge 92)  
+
+[!INCLUDE [contact DevTools team note](../../includes/edge-whats-new-note.md)]  
+
+
+> [!TIP]
+> The **Microsoft Build 2021** conference is during May 25-27.  Here's a video from Build about the updates to DevTools: [Microsoft Edge | State of the Platform][YoutubeEdgeStateOfThePlatform] - "Microsoft Edge brings a compelling and consistent platform and tools for developers.  As our legacy browsers phase out of support, Edge will soon be the only supported browser from Microsoft on Windows 10.  Join us to learn about the latest across the Edge platform, tools, and web apps."
+
+
+## The Close button is no longer hidden when DevTools is narrow
+
+<!-- Title: DevTools is now easier to close -->  
+<!-- Subtitle: The Close button in DevTools is always displayed, even when DevTools is docked to the right and the DevTools viewport is narrow. -->  
+
+In Microsoft Edge version 91 or earlier, the **Close** button to close DevTools isn't displayed when the DevTools viewport is narrow.  In Microsoft Edge version 92, the **Close** button in the DevTools is always present, regardless of the DevTools viewport width.
+
+:::image type="complex" source="../../media/2021/05/close-devtools-button-always-displayed.msft.png" alt-text="The Close DevTools button is now present even when the viewport is narrow" lightbox="../../media/2021/05/close-devtools-button-always-displayed.msft.png":::
+   The **Close** DevTools button is now present even when the viewport is narrow  
+:::image-end:::  
+
+
+## Add tools quickly with the new More Tools button
+
+<!-- Title: Add tools quickly with new More Tools button -->  
+<!-- Subtitle: A convenient way to open new tools in Microsoft Edge DevTools. -->  
+
+There's a new way to open more tools in Microsoft Edge DevTools: the **More Tools** (`+`) button and menu.  The **More Tools** (`+`) button appears on the upper toolbar and also on the lower toolbar (the toolbar of the drawer).  Selecting a tool from the **More Tools** menu adds the tool to the toolbar.
+
+To reorder the tool tabs on either toolbar, select and drag the tool tabs.
+
+:::image type="complex" source="../../media/2021/05/more-tools-button.msft.png" alt-text="The More Tools button" lightbox="../../media/2021/05/more-tools-button.msft.png":::
```

Comment:
It would be nice to show the + on the drawer toolbar

Reply:
Updated the capture in local working branch.

Reply:
This "More Tools" scenario, with two + buttons, is simpler than the wavy underlines.  I just finished re-doing this single capture to show both + signs, works well.